interface/__main__.py

```diff
@@ -19,7 +19,11 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 import time
 from threading import Thread
-from dynaconf import settings
+from dynaconf import Dynaconf
+
+settings = Dynaconf(
```

Comment:
As mentioned in matrix, loading `Dynaconf` in every module can add filesystem overheads.

Loading it once and re-exporting it should solve this issue.

`itnerface/settings.py`:

```python
from dynaconf import Dynaconf

settings = Dynaconf(
    environments=True, settings_files=["settings.toml"]
)
```

to use `interface.settings.py`, the following change is recommended:
`interface/ns.py` diff
```diff
diff --git a/interface/ns.py b/interface/ns.py
index ea8ab1f..d3afec1 100644
--- a/interface/ns.py
+++ b/interface/ns.py
@@ -21,7 +21,7 @@ from functools import lru_cache

 import requests

-from dynaconf import settings
+from interface.settings import settings
 from interface.utils import clean_url, since_epoch, trim_url
 from interface.error import Error
```